web/src/pages/bootcamp/bootcamp.module.scss

```diff
@@ -71,6 +71,10 @@
     }
 }
 
+html, body {
+    max-width: 100%;
+    overflow-x: hidden;
+}
 
```

Comment:
@ethangambles Do you know what this code block is for? I think this is causing the 5 across scrolling issue? 

I'm not sure if this effects anything on the bootcamp page, but we need to get rid of this because it's a "global" change. 

Can you remove this and confirm the bootcamp page is okay with this change? You can go ahead and merge the change once that is addressed.

Reply:
I added that because the bootcamp page has a horizontal scroll (You can see this on the live dev site). I didn't realize it was global. I also don't know why it's happening on the bootcamp page. I can try to fix it without making it a global change

Reply:
@raney24 I removed the `html` part of that statement and it seemed to fix both issues. It should be committed.